### PR TITLE
Update README to remove eclipse references.

### LINKED ISSUE #579: Read me needs update for 2019
Readme.md references eclipse and 2018 tool locations

**To Reproduce**  
1. Read Readme.md
2. Find references to eclipse and 2018 tool locations. 

**Expected behavior**  
1. Read Readme.md
2. Find references to vscode and frc2019 tool location

### PR DESCRIPTION
**Pull request by sciencewhiz**
Fixes #579

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,14 @@ subprojects that the main app depends on to provide data types, widgets, and dat
...
-It can also be run manually `java -jar C:\Users\<username\>wpilib\tools\Shuffleboard.jar`
+Shuffleboard is installed by the FRC vscode extension. It can be launched from the WPILib menu in vscode (start tool).
+It can also be run manually by running the shuffleboard.vbs in `c:\Users\public\frc<year>\tools` (Windows) or
+shuffleboard.py in ~/frc<year>/tools (Linux or Mac).
 
 ### Requirements
 - [JRE 11](http://jdk.java.net/11/). Java 11 is required.
 No other version of Java is supported. Java 11 is installed on Windows by the
-[FRC vscode extension](https://github.com/wpilibsuite/vscode-wpilib). Users on Mac or Linux will have to install Java 11
+[WPILib installer](https://github.com/wpilibsuite/allwpilib/releases). Users on Mac or Linux will have to install Java 11
```
Comment (SamCarlberg) [inline README.md]: Should we link to the screensteps docs on the WPILib installer and manual installation guides?

Comment (sciencewhiz) [reply README.md]: I think that's better in the installation section, what do you think?

Comment (SamCarlberg) [reply README.md]: That sounds good. Maybe add that section above "Running"?

O/T: I should probably move the "structure" section, but that can be done separately

File: README.md [hunk 2]
```diff
@@ -11,13 +11,16 @@ subprojects that the main app depends on to provide data types, widgets, and dat
 
 ## Running
 
-Shuffleboard is installed by the WPILib Eclipse Plugins. It can be launched from the WPILib menu in Eclipse.
-It can also be run manually `java -jar C:\Users\<username\>wpilib\tools\Shuffleboard.jar`
+Shuffleboard is installed by the FRC vscode extension: [Installation Instructions]
```
Comment (SamCarlberg) [inline README.md]: There can't be a newline or other gap between the `]` and `(` for the link to work

File: README.md [hunk 3]
```diff
@@ -11,13 +11,15 @@ subprojects that the main app depends on to provide data types, widgets, and dat
 
 ## Running
 
-Shuffleboard is installed by the WPILib Eclipse Plugins. It can be launched from the WPILib menu in Eclipse.
-It can also be run manually `java -jar C:\Users\<username\>wpilib\tools\Shuffleboard.jar`
+Shuffleboard is installed by the FRC vscode extension: [Installation Instructions](http://wpilib.screenstepslive.com/s/currentCS/m/java/l/1027503-installing-c-and-java-development-tools-for-frc).
+It can be launched from the WPILib menu in vscode (start tool).
```
Comment (AustinShalit) [inline README.md]:
```suggestion
It can be launched from the WPILib menu in Visual Studio Code (start tool).
```

**Pull request merged**